plankton/adapters/__init__.py

```diff
@@ -133,6 +138,12 @@ def __init__(self, target_member, property_name):
         self._target_member = target_member
         self._prop = property_name
 
+        try:
+            self.__doc__ = getattr(type(getattr(instance, self._target_member)),
+                                   self._prop).__doc__
+        except AttributeError:
+            pass
```

Comment:
No, because a `ForwardProperty` sometimes wraps a plain member of the device instance (for example `chopper.speed`). In that case `getattr(type(getattr(...)))` fails and no documentation is forwarded. I think that's okay if we want to continue allowing plain data members on devices.